src/uploads/generateFileData.ts

```diff
@@ -71,6 +71,7 @@ export const generateFileData = async <T>({
   let newData = data;
   const filesToSave: FileToSave[] = [];
   const fileData: Partial<FileData> = {};
+  const fileIsAnimated = (file.mimetype === 'image/gif') || (file.mimetype === 'image/webp');
```

Comment:
Not all `webp` media will be animated, is there a way to target and omit these cases?

Reply:
nope, not easily... adding `animated: true` should not have any negative repercussions either so i think we're good 👍